pkg/arbuilder/arbuilder.go

```diff
@@ -112,8 +112,6 @@ func (a *ArBuilder) getResults() ([]*service.FileNode, error) {
 			inputFileNode := &service.FileNode{}
 			ext := filepath.Ext(inFile)
 			switch ext {
-			case ".o":
```

Comment:
why is this deleted?

Reply:
@hemarkus It's included in the default case.